doc/FAQ.rst

```diff
@@ -1,6 +1,17 @@
 FAQ
 ===
 
+**Q: Where can I get help with pexpect?  Is there a mailing list?**
+
+A: As of 2015, the best ways to get general python support is the website
+forum stackoverflow_, the python-list_ mailing list, and the `#python`_
```

Comment:
This all seems very generic. I'd like to point to some slightly more specific things, like the [pexpect tag on SO](http://stackoverflow.com/questions/tagged/pexpect).